User_Stories.md

```diff
@@ -0,0 +1,11 @@
+﻿## User Stories
+
+As an end user, I want to be able to upload shapes or symbols for the artificial intelligence to detect so that I may create new targets on the fly.
+
+As an end user, I want to be able to select a symbol from a list of symbols so that the artificial intelligence may target the symbol with the laser.
+
+As an end user, I want the artificial intelligence to detect only ***balloons*** that have symbols on them so that random objects do not get burnt.
+
+As an end user, I want to be able to activate and deactivate the laser so that I can avoid burning holes in things that should not have a hole in them.
+
+As an end user, I want to be able to input commands so that I can move the laser in any direction I want.
```

Comment:
Didn't mention this in outside discussion but I feel like this should be removed, I don't see a valid use case for the user to have control over the laser except in the previous case of turning it off with the cancel button.